Consul/Agent.cs

```diff
@@ -1164,11 +1164,27 @@ public async Task<QueryResult<ServiceConfiguration>> GetServiceConfiguration(str
             return await _client.Get<ServiceConfiguration>($"/v1/agent/service/{serviceId}", q).Execute(ct).ConfigureAwait(false);
         }
 
+        /// <summary>
+        /// GetCARoots returns root certificates in the cluster
+        /// </summary>
+        /// <param name="ct">Cancellation Token</param>
+        /// <returns>Root certificates</returns>
         public async Task<QueryResult<CARoots>> GetCARoots(CancellationToken ct = default)
         {
             return await _client.Get<CARoots>("v1/agent/connect/ca/roots", QueryOptions.Default).Execute(ct).ConfigureAwait(false);
```

Comment:
Can you call the 
```suggestion
            return await GetCARoots(QueryOptions.Default, ct);
```

Reply:
Oh, makes sense. Shouldn't it be "return await GetCARoots(QueryOptions.Default, ct).ConfigureAwait(false);" though?

Reply:
yeah, you are right.